src/deadline/job_attachments/models.py

```diff
@@ -330,6 +330,30 @@ def partial_session_action_manifest_prefix(
             f"{_float_to_iso_datetime_string(time)}_{session_action_id}",
         )
 
+    @staticmethod
+    def partial_session_action_manifest_prefix_without_task(
```

Comment:
WorkerAgent would need this to store chunked session actions in a path without task id.